Collector/ProfilePlugin.php

```diff
@@ -58,7 +58,7 @@ public function handleRequest(RequestInterface $request, callable $next, callabl
     {
         $profile = new Profile($this->pluginName, $this->formatter->formatRequest($request));
 
-        if ($stack = $this->collector->getCurrentStack()) {
+        if (null !== $stack = $this->collector->getCurrentStack()) {
```

Comment:
Maybe this line will be easier to read if the assignation is moved out of the condition.